test/generator/user_context_insanity_aottest.cpp

```diff
@@ -39,10 +39,13 @@ int main(int argc, char **argv) {
 
     // Hijack halide's runtime to run a bunch of instances of this function
     // in parallel.
-    halide_do_par_for(nullptr, launcher_task, 0, num_launcher_tasks, nullptr);
+    int result = halide_do_par_for(nullptr, launcher_task, 0, num_launcher_tasks, nullptr);
```

Comment:
Same concern applies here. There's no way for this call to return a non-zero value.

Reply:
Commented.